lib/uploadcare/entity/file.rb

```diff
@@ -9,7 +9,8 @@ class File < Entity
       client_service FileClient
 
       attr_entity :datetime_removed, :datetime_stored, :datetime_uploaded, :image_info, :is_image, :is_ready,
-                  :mime_type, :original_file_url, :original_filename, :size, :url, :uuid
+                  :mime_type, :original_file_url, :original_filename, :size, :url, :uuid, :variations, :video_info,
+                  :source, :rekognition_info
```

Comment:
Maybe, `recognition_info`?

Reply:
it's an aws service name, so we just use it here verbatim